### PR TITLE
Restructure CWS best practices

### PR DESCRIPTION
**Pull request by AmySteam**
Fixes: https://github.com/GoogleChromeLabs/Extension-Docs/issues/5

## Description 

Slight restructure and update of the CWS best practices:

- Changed headers and subheaders
- Added descriptions to categories

That's about it :)

### CODE REVIEW

File: site/en/docs/webstore/best_practices/index.md [hunk 1]
```diff
@@ -1,117 +1,133 @@
...
+
+Design your extension with the user in mind by providing a simple, intuitive, and seamless user interface while also respecting user privacy.
+
+### Onboarding experience
+
+Start onboarding your users as soon as they reach your store listing by providing screenshots and a video of how the extension works. We recommend following the [permission warning guidelines][doc-perm-warn] to increase the chances of users installing your extension.
+
+### Designing a persistent UI
+
+When designing a [side panel][api-sidepanel] for your extension, make sure it enhances the user's browsing experience by providing relevant information and useful functionality. A side panel should help users accomplish tasks with as little distraction as possible.
```
[the remark this answers is not in the document]
Comment (AmySteam) [reply site/en/docs/webstore/best_practices/index.md:47]: The wording was requested by David/Alex to match the [quality guidelines](https://developer.chrome.com/docs/webstore/program-policies/quality-guidelines/#:~:text=If%20utilizing%20a%20persistent%20user%20interface%2C%20extensions%20should%20actively%20enhance%20the%20user%27s%20current%20task%20while%20causing%20minimal%20distractions.) 

I'm open to another suggestion... 🤷🏻‍♀️

File: site/en/docs/webstore/best_practices/index.md [hunk 2]
```diff
@@ -1,117 +1,133 @@
...
+[oauth2-tutorial]: /docs/extensions/mv3/tut_oauth/
+[privacy-policy]: /docs/webstore/publish/#setup-a-developer-account
+[privacy-tab]: /docs/webstore/cws-dashboard-privacy/#certify-your-data-use-practices
+[program-policies]: /docs/webstore/program-policies/
+[puppeteer]: https://pptr.dev/guides/chrome-extensions
+[single-purpose]: /docs/extensions/mv3/single_purpose/
 [stay-secure]: /docs/extensions/mv3/security/
 [supplying-images]: /docs/webstore/images/
 [user-data-faq]: /docs/webstore/user_data/
+[user-data]: /docs/webstore/user_data/
```
[the remark this answers is not in the document]
Comment (AmySteam) [reply site/en/docs/webstore/best_practices/index.md:132]: 😶‍🌫️

File: site/en/docs/webstore/best_practices/index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (sebastianbenz) [reply site/en/docs/webstore/best_practices/index.md:47]:
```suggestion
Avoid distracting users when implementing a persistent UI. For example, when designing a [side panel][api-sidepanel] for your extension, make sure it enhances the user's browsing experience by providing relevant information and useful functionality. A side panel should help users accomplish tasks with as little distraction as possible.
```

**Review by sebastianbenz — APPROVED** [review]

File: site/en/docs/webstore/best_practices/index.md [hunk 3]
```diff
@@ -1,117 +1,130 @@
...
 
-Privacy
-: Make sure that your extension handles user data appropriately and conforms to [Chrome
-Web Store's data privacy](/docs/webstore/program-policies/privacy) requirements. (See this [FAQ][user-data-faq] for further details.)
+## Security
 
-Performance
-: Does the extension function at an outstanding level? High-quality extensions don't
-just perform their intended action, they do so while using as few system resources as possible.
+Your extension should be safe for your users. For example, transmit user data securely via HTTPS or WSS. Stored user data must be strongly encrypted using RSA or AES. Check that your extension does not pose security threats and does not use [deceptive installation tactics][deceptive-install]. See [Stay secure][stay-secure] for a more in-depth discussion.
```
Comment (AmySteam) [inline site/en/docs/webstore/best_practices/index.md:25]:
```suggestion
Your extension should be safe for your users. For example, send user data securely via HTTPS or WSS. Check that your extension does not pose security threats and does not use [deceptive installation tactics][deceptive-install]. See [Stay secure][stay-secure] for a more in-depth discussion.
```
Add in another iteration

**Comment by AmySteam** [comment]
@jpmedley This is ready for an editorial review. Thanks! 🙏

**Comment by AmySteam** [comment]
@jpmedley Can you take a quick look?

**Review by jpmedley — CHANGES_REQUESTED** [review]

File: site/en/docs/webstore/best_practices/index.md [hunk 4]
```diff
@@ -1,117 +1,130 @@
...
-updated: 2023-06-09
+updated: 2023-08-07
 description: >
-  Best practices on how to implement your extension and list it in the Chrome Web Store.
+  How to create a high-quality extension and Chrome Web store listing.
 ---
 
-This page has advice on how you should implement your extension and list it in the store. As the store
-matures and we learn from developers' experiences, these recommendations will be updated.
+## Overview {: #design-a-high-quality-extension}
```
[the remark this answers is not in the document]
Comment (AmySteam) [reply site/en/docs/webstore/best_practices/index.md:11]: Totally get your point. However, I'm keeping the header-id to ensure any links to this section still work.

**Review by jpmedley — APPROVED** [review]

**Pull request merged**